website/src/components/examples.js

```diff
@@ -32,15 +26,22 @@ export default class Examples extends Component {
     window.onresize = null;
   }
 
+  _getSize() {
+    const {innerWidth, innerHeight} = window;
+
+    return {
+      width: innerWidth >= 576 ? innerWidth - WIDTH_OFFSET : innerWidth,
```

Comment:
> Not sure if there are better ways to manage this variable other than hard coding it.

A reasonable way to deal with hard coded numbers is to assign them to constants with descriptive names. Either at the top of the file or directly in the function. That gives the next programmer to work on this a little more context about what is going on.